content/chapters/software-stack/lab/content/libcall-syscall.md

```diff
@@ -49,6 +49,14 @@ The main reason for `fwrite()` not making any system calls is the use of a stand
 Calls the `fwrite()` end up writing to that buffer to reduce the number of system calls.
 Actual system calls are made either when the standard C library buffer is full or when an `fflush()` library call is made.
 
+Note that on some systems, `ltrace` **does not work** as expected, due to `now binding`.
+To avoid this behaviour, you can force the `lazy binding` (based on which `ltrace` is constructed to work).
+An example can be found in `support/libcall-syscall/Makefile`, however for system binaries, such as `ls` or `pwd`, the only alternative is:
+
+```console
+student@os:~$ ltrace -x "*"
+```
+
```

Comment:
I would add some text and a command-line snippet saying that if one only wants to trace certain known symbols, they can use `-x "malloc" -x "free"` etc.